### ISSUE TITLE
Game request: Cafe Stella to Shinigami no Chou

### ISSUE THREAD

**Issue by Veshurik** [morkt/GARbro#396]
VNDB: https://vndb.org/v26414
Engine: Kirikiri

Seems like every new Yuzusoft game has unique encryption, and no one program doesn't help (KrKrExtract, kikiriki etc).
I really don't know what to do, but I hope that someone will dig into files, because Yuzusoft is really popular. Thanks!

**Comment by jszhtian** [comment]
Krkrextract 4.0.1.5 can decryp the Xp3 package.
Actually, the protection scheme is the same, but key is different.
You can also write a dll to dump files like normal krkrz games.

**Comment by Veshurik** [comment]
I'm zero in programming, huh...

But I trying 4.0.1.5, and it doesn't work.

I mean, on my PC "drap and drop .exe here" never worked. 
I don't even understand why.

And other way (renaming KrKrExtract.dll to .tpm and pasting into "plugin" folder, then run cmd and type "cafestella.exe -nosel -graphicsystem=drawdevice ) also doesn't work. 

-_-

**Comment by Cerzs** [comment]
@Veshurik Try drag and drop the game exe into the krkrExtract.exe

**Comment by jszhtian** [comment]
If you login system with built-in Administrator account or use "Run as Administrator", Some application's drap and drop function maybe not work.
You need a DLL injector to inject KrkrExtract.dll to main process or rebuild CafeStella.exe to modify its IAT.
Maybe Detours Tools can help you.
[setdll.zip](https://github.com/morkt/GARbro/files/3989916/setdll.zip)

**Comment by Veshurik** [comment]
If you can write a manual, how to use all this, it would be great.
Because no one are going to extract all files, huh.

**Comment by Veshurik** [comment]
Also, if I find a game, which works, program crashes, when I starting extract files with System (PNG) setting. If I use "RAW", then it's ok, but when System PNG...
If I choose just PNG, files extracts not with alpha channel (like JPG), and that's strange...

**Comment by Veshurik** [comment]
Ehm. I finally understood, how to use that program.

1. Just drag CafeStella.exe on KrKrExtract.exe, don't run the program!!!
2. DON'T TOUCH ANYHOW GAME WINDOW!!! Otherwise the program crashes.

No one on forums talks about that. So, I want to share it with everyone else. Incredible.

But sometimes it doesn't work, for example, for Cross Concerto (https://vndb.org/v21647), when crack for the game uses another .exe... And I don't know what to do with that.

**Comment by MishaIac** [comment]
I know how easy it is to decrypt game files (in the sense that the game will perceive the file "patch.xp3" packaged through GARbro) so that I can do it the same way I did with Riddle Joker as well. True, I have a problem, I do not have the game itself, so I can’t help you now ¯ \ _ (ツ) _ / ¯
I’ll wait until they upload a torrent tracker.
p.s. Sorry for my English. Actually, I'm a Russian guy, so I use a translator.

**Comment by Veshurik** [comment]
Я тоже русский.
На самом деле, я уже извлёк файлы из этой игры, все нормально.

Но с другими играми, как например, Cross Concerto, это не прокатывает. 
Ещё также и с другими играми.
Я просто веду группу, где все экстракты выкладываю, и некоторые новеллы просто не могу из-за этого выложить, так как они зашифрованы (Tamayura Mirai, Cross Concerto и другие).

Если у вас есть какой-то инструмент или вы бы могли помочь, был бы благодарен.

**Comment by MishaIac** [comment]
> Я тоже русский.
> На самом деле, я уже извлёк файлы из этой игры, все нормально.
> 
> Но с другими играми, как например, Cross Concerto, это не прокатывает.
> Ещё также и с другими играми.
> Я просто веду группу, где все экстракты выкладываю, и некоторые новеллы просто не могу из-за этого выложить, так как они зашифрованы (Tamayura Mirai, Cross Concerto и другие).
> 
> Если у вас есть какой-то инструмент или вы бы могли помочь, был бы благодарен.

Да, можно легко извлечь, но когда вы обратно упакуете файлы в patch.xp3, игра просто не воспримет упакованные файлы. Насчет других игр не знаю, но если они на KiriKiri, то попробую извлечь (можем в дискорд пойти, там мне удобнее, если согласны, киньте свой ник и тэг).

**Comment by bai0012** [comment]
+1

**Comment by Veshurik** [comment]
Well, it works with KrKrExtract, but with GarBro is more comfortable to browse files

**Comment by Jackyes-jpg** [comment]
I hope to use an external program to open and edit the file (I am Russian, these are translations)

**Comment by JHPY** [comment]
Hello @Veshurik , thanks for posting a solution to the problem. However, I got another one: 
![1](https://user-images.githubusercontent.com/71465644/93520550-3be18200-f961-11ea-84e0-4887f809b088.png)

Can someone help? I already copied the all the extractor files to the game folder. Does this have something to do withe the antivirus?

**Comment by crskycode** [comment]
Use this mod [Link](https://github.com/crskycode/GARbro/releases)

**Comment by Lyoko-Jeremie** [comment]
the Krkrextract cannot extart BGM in my computer by unkown reason (other data file are extarted) , your mod help me, thank you~ @crskycode
